server/db/models/user.js

```diff
@@ -0,0 +1,183 @@
+const { Model, STRING, INTEGER, UUID, UUIDV4, BOOLEAN } = require('sequelize');
+const db = require('../connection');
+const {
+  makeHash,
+  titleCase,
+  compareStrAgainstHash,
+} = require('../../../utils/index');
+const { AuthenticationError } = require('../../../utils/backend');
+
+class User extends Model {}
+User.init(
+  {
+    id: {
+      type: UUID,
+      defaultValue: UUIDV4,
+      unique: true,
+      primaryKey: true,
+    },
+    firstName: {
+      type: STRING,
+      allowNull: false,
+      validate: {
+        notEmpty: true,
+      },
+    },
+    lastName: {
+      type: STRING,
+      allowNull: false,
+      validate: {
+        notEmpty: true,
+      },
+    },
```

Comment:
I'm kind of thinking that First and last name could be allowed to be null?
Where the username is the key element, not the actual username?